cheatsheet.html

```
@@ -1712,9 +1721,9 @@ <h3 class="subheading">Other utilities</h3>
 
 <footer class="footer-note">
 <<<<<<< HEAD
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)